ui/store/actions.js

```diff
@@ -3802,3 +3802,7 @@ export function requestAddNetworkApproval(customRpc, originIsMetaMask) {
     }
   };
 }
+
+export async function addEventBeforeMetricsOptIn(event) {
+  await promisifiedBackground.addEventBeforeMetricsOptIn(event);
+}
```

Comment:
Remove this. We likely will never have a need for this from the UI. You likely meant to as there isn't a corresponding change in metamask-controllers.